src/Illuminate/Queue/Events/WorkerStopping.php

```diff
@@ -15,5 +15,6 @@ public function __construct(
         public $status = 0,
         public $workerOptions = null
     ) {
+        $this->status = $status;
```

Comment:
Why do we need this? Isn't this property a promoted property already?

Reply:
Good catch 👍🏻 It's a leftover, I missed